backend/tests/Designer.Tests/Controllers/OptionsController/ImportOptionsListFromOrgTests.cs

```diff
@@ -0,0 +1,126 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Models;
+using Designer.Tests.Controllers.ApiTests;
+using Designer.Tests.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Designer.Tests.Controllers.OptionsController;
+
+public class ImportOptionsListFromOrgTests : DesignerEndpointsTestsBase<ImportOptionsListFromOrgTests>, IClassFixture<WebApplicationFactory<Program>>
+{
+    public ImportOptionsListFromOrgTests(WebApplicationFactory<Program> fixture) : base(fixture)
+    {
+    }
+
+    private const string OrgName = "ttd";
+    private const string Username = "testUser";
+
+    [Fact]
+    public async Task Post_Returns200OK_WhenImportingCodeListFromOrg()
+    {
+        // Arrange
+        const string orgRepoName = "org-content";
+        const string appRepoName = "empty-app";
+        const string optionListId = "codeListString";
+        const string stringCodeList = @"[
+            {""value"": ""norway"",""label"": ""Norge""},
+            {""value"": ""denmark"",""label"": ""Danmark""},
+            {""value"": ""sweden"",""label"": ""country.label.sweden""}
+        ]";
+        List<Option> expectedOptionList = JsonSerializer.Deserialize<List<Option>>(stringCodeList);
+
+        string targetOrg = TestDataHelper.GenerateTestOrgName();
+        string targetOrgRepository = TestDataHelper.GetOrgContentRepoName(targetOrg);
+        await CopyOrgRepositoryForTest(Username, OrgName, orgRepoName, targetOrg, targetOrgRepository);
+
+        string targetAppRepository = TestDataHelper.GenerateTestRepoName();
```

Comment:
Hva tenker du om å bruke `-Name` her også?
```suggestion
        string targetOrgName = TestDataHelper.GenerateTestOrgName();
        string targetOrgRepoName = TestDataHelper.GetOrgContentRepoName(targetOrg);
        await CopyOrgRepositoryForTest(Username, OrgName, orgRepoName, targetOrgName, targetOrgRepoName);

        string targetAppRepoName = TestDataHelper.GenerateTestRepoName();
```